queries/filter_expression_with_dot_notation_with_number/selector

```diff
@@ -0,0 +1 @@
+$[?(@['2'] == 'second')]
```

Comment:
Shouldn't this be called something like "filter expression with bracket notation with number", following e,g. https://cburgmer.github.io/json-path-comparison/#filter_expression_with_bracket_notation ?

Reply:
Rather, the folder name is as intended but the selector was wrong. I was experimenting with a few different selectors, and left the wrong one in the selector file.  It should be okay now.